### PR TITLE
feat: Homepage

### PR DESCRIPTION
**Pull request by manojncsoni007**
Description:
- Added home page
- Added Navbar
- created video context

Screenshot
Screenshot (278)

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    color: var(--black);
+}
+
+.greytext {
+    color: var(--blue);
+}
+
+.redtext {
+    color: var(--red);
+}
```
Comment (srishti-maurya) [inline src/components/Navbar/Navbar.css:12]: you may name them as `grey-text` and `red-text` for better readability

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,19 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar } from "./components";
+import { Home, VideoListing } from "./pages";
+import { Route, Router, Routes } from 'react-router-dom';
+// import { Home, VideoListing } from "./pages";
```
Comment (srishti-maurya) [inline src/App.js:5]: The commented line is already used in `line 3`, you may delete them from here

**Review by srishti-maurya — APPROVED** [review]
Looks good!
Liked the way you have used the barrel exports.
Suggestion: You may add some space in the footer for better UI

**Review by shashikantmaurya0007 — APPROVED** [review]
Looks good to merge

File: src/App.js (hunk 2 above)
Comment (manojncsoni007) [reply src/App.js:5]: okay, will do

File: src/components/Navbar/Navbar.css (hunk 1 above)
Comment (manojncsoni007) [reply src/components/Navbar/Navbar.css:12]: sure

**Review by sonishreyas — COMMENTED** [review]
Code looks good. Like the folder structure

**Pull request merged**